app/validators/return-versions/setup/start-date.validator.js

```diff
@@ -53,16 +53,24 @@ function _fullDate(payload) {
 }
 
 function _validateAnotherStartDate(payload, licenceStartDate, licenceEndDate) {
+  let minDate = licenceStartDate
+  let minMessage = 'Start date must be on or after the original licence start date'
+
+  if (new Date(licenceStartDate) < new Date('1959-04-01')) {
+    minDate = '1959-04-01'
+    minMessage = 'Start date must be on or after 1 April 1959'
+  }
```

Comment:
Could I ask that this be put into a function of its own? Perhaps `_minimumDateDetails()`, which returns `{ minDate, minMessage }`.

Not only does this move this 'noise' out of the validation method, it also gives us somewhere to add the same not we've used in the PR. So, should we come by in 6 months, we'll have an explanation for why 1959-04-01 is being used for this validation.